### ISSUE TITLE
Is there any plant to support multiple items to drag at once.

### ISSUE THREAD

**Issue by rrohitesh** [ObaidUrRehman/ng2-drag-drop#54]
Hello Obaid,

Is there any plan to implement the multiple select for drag item.

**Comment by gravisrinivas** [comment]
Any plans on implementing this is near future ?

**Comment by ObaidUrRehman** [comment]
Yes absolutely, this is planned for future after the sortable feat.

**Comment by gravisrinivas** [comment]
Is there any estimated period like 4months or 6 months?

**Comment by Usamanawaz** [comment]
@gravisrinivas  You can do this simply by sending a list as drag data and every time you click on an item it is added to the list.

**Comment by gravisrinivas** [comment]
@Usamanawaz : If i am collecting the items in a list individually , then how is it a multiple drag and drop . I should be able to select multiple items. Collecting individual item was never the question

**Comment by manishambashta** [comment]
@ObaidUrRehman : any updates on this enhancement??
